install/install.go

```diff
@@ -210,385 +218,93 @@ func checkTerminatingPods(c *k8s.Client, ns string) int {
 	return 0
 }
 
-// K8sInstaller for karmor install
-func K8sInstaller(c *k8s.Client, o Options) error {
```

Comment:
Let's not remove it entirely, let's keep it under the legacy flag. It's still convenient for quick testing where operator is not needed.

Reply:
Sounds good! I'll create a flag for legacy install and uninstall.

Reply:
@daemon1024 If a user installs using `--legacy true` then karmor uninstall will throw an error as legacy installation doesn't create helm release. To enforce legacy uninstall I see 2 options - 

1. Inform the user to try legacy uninstall using` karmor uninstall --legacy=true` when helm throws error.
2. When using legacy installation. Create a label on the daemonset `legacy=true` which will be used to automatically uninstall using the legacy approach.
**The second approach does complicate it a bit but automates the uninstallation while the first one is self-explanatory.
Eager to know your thoughts!**

Reply:
Let's go with first method. It's fine to do that.